core/src/main/java/com/expediagroup/streamplatform/streamregistry/core/services/ConsumerBindingService.java

```diff
@@ -15,11 +15,13 @@
  */
 package com.expediagroup.streamplatform.streamregistry.core.services;
 
+import static com.expediagroup.streamplatform.streamregistry.DataToModel.convertToModel;
```

Comment:
proper injected collaborators, rather than static utility methods, would simplify testing. Here you'd be able to pass mocks around to verify composition and leave the conversion testing to the converter class.

Reply:
Thats true but unit tests against DataToModel were written with no difficulty. If it is used in other classes, unit tests can still be written and the DataToModel functions have no dependencies.

Can you give an example if what would be solved. I agree in principal but don't see where it helps us in this case.

Reply:
it means you have to test the conversion twice - once in the converter itself and again where it's used. But we have no isolated units tests for `ConsumerBindingService` so I can't give you an example.

Reply:
ModelToData and DataToModel are both now injected Components